storage-model/src/main/java/no/unit/nva/publication/storage/model/DoiRequest.java

```diff
@@ -157,10 +182,67 @@ public String getStatusString() {
         return getStatus().toString();
     }
 
-    private SortableIdentifier validateResourceIdentifier(SortableIdentifier resourceIdentifier) {
-        if (nonNull(resourceIdentifier)) {
-            return resourceIdentifier;
+
+
+    public Publication toPublication() {
```

Comment:
Publication is the de facto DTO for entries in the table.  DoiRequests are for now mapped to a thin publication object that will duplicate the information that are necessary for updating a DOI or other purposes. For now it contains only the main title as it is the information the information that frontend needs when listing the DOIs.